SPECS/compiler-rt/compiler-rt.spec

```diff
@@ -30,9 +30,10 @@ BlocksRuntime is an implementation of Apple "blocks" interface.
 %build
 mkdir -p build
 cd build
-%cmake -DCMAKE_BUILD_TYPE=Release  \
-	-DCOMPILER_RT_INSTALL_PATH=%{_prefix}/lib/clang/%{maj_ver} \
-       -Wno-dev ../compiler-rt
+%cmake \
+        -DCMAKE_BUILD_TYPE=Release  \
+        -DCOMPILER_RT_INSTALL_PATH=%{_prefix}/lib/clang/%{maj_ver} \
```

Comment:
Is it a typo, as I do not see a change in the install path, only whitespace change

Reply:
It's the maj_ver macro that was problematic.  But I agree it seems like no other changes in this are of code beside whitespace

Reply:
yes, the fix is to `maj_ver` which should be "18" instead of "18.1.2". The change to "%cmake" is just cosmetic